packages/ckeditor5-dev-webpack-plugin/lib/replacetranslationcallsforonelanguage.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const {
+	getAllTranslations,
+	replaceFirstFunctionParameter,
+	createTranslator
+} = require( '@ckeditor/ckeditor5-dev-utils' );
+
+/**
+ * Replaces all function call parameters with translated strings for the t function.
+ *
+ * @param {Object} compiler Webpack compiler.
+ * @param {String} language Language code, e.g en_US.
+ */
+module.exports = function replaceTranslationCallsForOneLangauge( compiler, language ) {
+	const packageNames = new Set();
+
+	compiler.plugin( 'after-resolvers', ( compiler ) => {
+		compiler.resolvers.normal.plugin( 'before-resolve', ( obj, done ) => {
+			const match = obj.request.match( /@ckeditor[\\\/]([^\\\/]+)/ );
+
+			if ( match ) {
+				packageNames.add( match[ 1 ] );
+			}
+
+			done();
+		} );
+	} );
+
+	compiler.plugin( 'emit', ( options, done ) => {
+		const allTranslations = getAllTranslations( packageNames, language );
```

Comment:
The `getAllTranslations()` function cannot hardcode how the path to package folders is resolved. How do you know how Webpack has been configured? It should use the paths which Webpack uses (so resolved by Webpack).